cypress/features/regression/confirm.feature

```diff
@@ -1,154 +1,86 @@
+      | mp150ú¿¡üßä             | titleOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{} | titleSpecialCharacter |
   # @ignore because of FE-2782
   # | &"'<>|
 
   @positive
   Scenario Outline: Change subtitle in Confirm dialog to <subtitle>
-    When I set subtitle to <subtitle> word
-      And I open component preview
+    When I open default "Confirm" component in noIFrame with "confirm" json from "commonComponents" using "<nameOfObject>" object name
     Then dialog subtitle context is <subtitle>
     Examples:
-      | subtitle                |
-      | mp150ú¿¡üßä             |
-      | !@#$%^*()_+-=~[];:.,?{} |
+      | subtitle                | nameOfObject             |
+      | mp150ú¿¡üßä             | subtitleOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{} | subtitleSpecialCharacter |
   # @ignore because of FE-2782
   # | &"'<>|
 
   @positive
   Scenario Outline: Change the height of Confirm dialog to <height>
-    When I set height to "<height>"
-      And I open component preview
+    When I open default "Confirm" component in noIFrame with "confirm" json from "commonComponents" using "<nameOfObject>" object name
     Then Confirm dialog input height is "<height>"
     Examples:
-      | height |
-      | 0      |
-      | 10     |
-      | 999    |
-      | 1500   |
+      | height | nameOfObject |
+      | 0      | height0      |
+      | 10     | height10     |
+      | 999    | height999    |
+      | 1500   | height1500   |
 
   @positive
   Scenario Outline: Change the size of Confirm dialog to <sizeName>
-    When I select size to "<sizeName>"
-      And I open component preview
+    When I open default "Confirm" component in noIFrame with "confirm" json from "commonComponents" using "<nameOfObject>" object name
     Then Confirm dialog size property on preview is "<sizePropertyInPx>"
     Examples:
-      | sizeName     | sizePropertyInPx |
-      | extra-small  | 300              |
-      | small        | 380              |
-      | medium-small | 540              |
-      | medium       | 750              |
-      | medium-large | 850              |
-      | large        | 960              |
-      | extra-large  | 1080             |
+      | sizeName     | sizePropertyInPx | nameOfObject |
+      | extra-small  | 300              | sizeExtraSmall |
+      | small        | 380              | sizeSmall |
+      | medium-small | 540              | sizeMediumSmall |
+      | medium       | 750              | sizeMedium |
+      | medium-large | 850              | sizeMediumLarge |
+      | large        | 960              | sizeLarge |
+      | extra-large  | 1080             | sizeExtraLarge |
 
   @positive
   Scenario: Enable background UI
-    When I check enableBackgroundUI checkbox
-      And I open component preview
-    Then Background UI is enabled
+    When I open default "Confirm" component in noIFrame with "confirm" json from "commonComponents" using "enableBackgroundUI" object name
```

Comment:
```suggestion
    When I open default "Confirm" component in noIFrame with "confirm" json from "commonComponents" using "disabledBackgroundUI" object name
```

Reply:
I'm going with convention for the naming: `${knob}${value}`